docs/sources/installation/troubleshooting.md

```diff
@@ -30,7 +30,59 @@ If you encounter an error or problem it is a good idea to check the grafana serv
 located at `/var/log/grafana/grafana.log` on Unix systems or in `<grafana_install_dir>/data/log` on
 other platforms and manual installs.
 
-You can enable more logging by changing log level in you grafana configuration file.
+You can enable more logging by changing log level in your grafana configuration file.
+
+## Diagnostics
+
+The `grafana-server` process can be instructued to enable certain diagnostics when it starts. This can be helpful
+when experiencing/investigating certain performance problems. It's `not` recommended to have these enabled per default.
+
+### Profiling
+
+The `grafana-server` can be started with the arguments `-profile` to enable profiling and  `-profile-port` to override
+the default HTTP port (`6060`) where the pprof debugging endpoints will be available, e.g.
+
+```bash
+./grafana-server -profile -profile-port=8080
```

Comment:
Using port `8080` as an example port instead of `6060` is it intentional for highlighting the fact that the default port can be overridden?

Reply:
Yes